### ISSUE TITLE
Externals normalization

### ISSUE THREAD

**Issue by guybedford** [rollup/rollup#633]
With the new normalization changes on 0.26, I'm getting a couple of bugs with externals normalization.

main.js
```javascript
import {fn} from './dep.js';
fn();
```

dep.js
```javascript
export function fn(){...}
```

The Rollup build is then done with:

```javascript
    return rollup.rollup({
      entry: 'main.js',
      external: ['dep.js'],
      plugins: [{    
        resolveId: function(id, importer, options) {
          if (id == './dep.js')
            return 'dep.js';
          return id;
        },
        load: function(id, options) {
          return sources[id];
        }
      }],
      onwarn: function(message) {}
    })
```

The output file then contains:

```javascript
import { fn } from '../dep.js';
fn();
```

Where the `../dep.js` has been added here incorrectly.

**Comment by Rich-Harris** [comment]
Apologies for the long wait... this is fixed in 0.26.4

**Comment by guybedford** [comment]
Thanks so much, SystemJS Builder finally works on 0.26.4!!
